### PR TITLE
Placeholders are not displayed in IE

### PR DESCRIPTION
**Pull request by Coenego**
The placeholders for the login form are not displayed in IE. (Low priority as this will not be used in a production environment)

### CODE REVIEW

**Comment by Coenego** [comment]
Assigning to @bp323 for review

**Comment by bp323** [comment]
Can you also apply this to the batch edit header input fields?

**Comment by bp323** [comment]
Back to @Coenego for followup

**Comment by Coenego** [comment]
Back to @bp323 for review

File: shared/gh/js/views/tenant-admin/gh.admin-batch-edit.js [hunk 1]
```diff
@@ -1450,7 +1451,17 @@ define(['gh.core', 'gh.constants', 'moment', 'moment-timezone', 'gh.calendar', '
...
+        $(document).on('gh.batchedit.rendered', function() {
+
+            // Set up the placeholders for input fields
+            $('#gh-batch-edit-header').onAvailable(function() {
+                $('#gh-batch-edit-title').placeholder();
+                $('#gh-batch-edit-location').placeholder();
+            });
+
+            // Set up the jEditable components
+            setUpJEditable
```
Comment (bp323) [inline shared/gh/js/views/tenant-admin/gh.admin-batch-edit.js]: `setUpJEditable();`

File: shared/gh/js/views/tenant-admin/gh.new-series.js [hunk 2]
```diff
@@ -155,5 +155,25 @@ define(['gh.core', 'gh.constants', 'gh.api.orgunit', 'gh.api.series', 'gh.api.or
...
+
+    /**
+     * Initialise the new series functionality
+     * @private
+     */
+    var initialise = function() {
+
+        // Add the placeholder to the input field (IE fix)
+        $('#gh-new-series-form').find('#gh-series-name').onAvailable(function() {
+            $('#gh-series-name').placeholder();
```
Comment (bp323) [inline shared/gh/js/views/tenant-admin/gh.new-series.js]: Needs a `.focus()` at the end

**Comment by bp323** [comment]
A few minor issues remaining + tests are failing. Reassigning to myself to followup as @Coenego is OOTO.

**Comment by bp323** [comment]
Followed up and merged into master

**Pull request merged**